README.md

```diff
@@ -1,36 +1,53 @@
+[![Build Status](https://travis-ci.org/01org/libyami.svg?branch=apache)](https://travis-ci.org/01org/libyami)
+[![Coverity Scan Build Status](https://scan.coverity.com/projects/11605/badge.svg)](https://scan.coverity.com/projects/01org-libyami)
+
 libyami
+-------
 Yet Another Media Infrastructure.
+
 It is YUMMY to your video experience on Linux like platform.
 
+
 Yami is core building block for media solution. it parses video stream
+
 and decodes them leverage hardware acceleration.
 
 
 Copyright (c) 2010, The WebM Project authors.
+
 Copyright (C) 2011-2016 Intel Corporation
+
 Copyright (C) 2015-2016 Alibaba
 
 
 License
 -------
 libyami libraries are available under the terms of the
+
 Apache License 2.0
 
 
 Overview
 --------
 libyami consists of several libraries:
+
     * 'codecparsers': it is bit stream parser,
+
```

Comment:
seems this is little illformatted. https://github.com/wudping/libyami/tree/TravicCI